Lib/collections/__init__.py

```diff
@@ -961,6 +961,24 @@ def clear(self):
         'Clear maps[0], leaving maps[1:] intact.'
         self.maps[0].clear()
 
+    def __ior__(self, other):
+        self.maps[0].update(other)
+        return self
+
+    def __or__(self, other):
+        if isinstance(other, _collections_abc.Mapping):
+            m = self.maps[0].copy()
+            m.update(other)
+            return self.__class__(m, *self.maps[1:])
+        return NotImplemented
+
+    def __ror__(self, other):
+        if isinstance(other, _collections_abc.MutableMapping):
+            m = other.copy()
```

Comment:
Here you go, @curtisbucher:

```suggestion
        if isinstance(other, _collections_abc.Mapping):
            m = dict(other)
```

Reply:
Ah, we crossed posts.